templates/results-table.php

```diff
@@ -0,0 +1,32 @@
+<script type="text/template" id="tmpl-plugin-check-results-table">
+	<table id="plugin-check__results-table-{{data.index}}" class="plugin-check__results-table">
```

Comment:
```suggestion
	<table id="plugin-check__results-table-{{data.index}}" class="widefat plugin-check__results-table">
```

Reply:
Thanks @spacedmonkey updated this with your suggestion